Marlin/Marlin_main.cpp

```diff
@@ -117,7 +119,8 @@
  * M100 - Watch Free Memory (for debugging) (Requires M100_FREE_MEMORY_WATCHER)
  * M104 - Set extruder target temp.
  * M105 - Report current temperatures.
- * M106 - Fan on.
+ * M106 - Fan on: S0-255 : T3-255 / T2 / T1 Temporary set speed during printing
+ *        T1 return / T2 apply / T3-255 set without applying
```

Comment:
Move this kind of documentation to the comment above `gcode_M106`. These descriptions are intentionally terse.